<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nume</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lastname_box"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Prenume</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="firstname_box"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Nr. telefon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="phone_box"/>
     </item>
     <item>
      <widget class="QPushButton" name="search_button">
       <property name="text">
        <string>Cauta</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
